---
name: Conor M.
surname: Finlay
position: "Senior Research Fellow"
address: "Trinity Translational Medicine Institute, Trinity College Dublin"
phone: +353 857148825
profilepic: Conor.jpeg
#www: "github.com/Conorisco"
email: "cofinlay@tcd.ie; conor.finlay@manchester.ac.uk"
twitter: conorisco
github: conorisco
googlescholar: kQ6U7AkAAAAJ
#headcolor:
orcid: 0000-0001-8285-0903
#linkedin: mariecurie
date: "`r format(Sys.time(), '%B %Y')`"
output: vitae::awesomecv 
#output: vitae::twentyseconds
aboutme: "Cellular Immunologist and bioinformatician combinding molecular, in vivo and translational research"
#docname: "Conor Finlay CV.pdf"
#headcolor: 	050583
fontsize: 10pt # be aware thea LaTeX has limitations and cannot make text less than 10pt
headcolor: 224190
---

<!-- #\fontsize{11}{18}{8}{8} -->

```{r install packages, include=FALSE}
#install.packages("tidyverse")
#install.packages("vitae")
#install.packages("rorcid")
#install.packages('scholar')
#install.packages("readxl")
#install.packages("knitr")
#install.packages('tinytex')
#install.packages("gridExtra")
#tinytex::install_tinytex() # installs LaTeX - some issues on windows (but only some)
# knitting for first time will take a while as it appears to install a bunch of tools for latex on run 1
```

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
pacs <- c('vitae','tidyverse','readr','readxl','rorcid','rmarkdown','knitr','scholar','cowplot','gridExtra','patchwork')
lapply(pacs, library, character.only=TRUE) #%>% invisible()
myor <- "0000-0001-8285-0903"
myscol <- 'kQ6U7AkAAAAJ'
col <- "#050583"
```

```{r google scholar, include=FALSE}
l <- get_profile(myscol)
p <- get_publications(myscol)
ct <- get_citation_history(myscol)
```

<!-- # Research interests  -->

<!-- Cellular Immunologist with three core skills: *in vivo* **models of inflammation, bioinformatics and clinical research**. My main research interest is in what factors control mononuclear phagocyte (macrophage) differentiation during inflammation on a single cell level and how this impacts on disease outcomes. I have expertise T cell function, autoimmunity, type 2 immune responses, stromal-immune cell interactions, myeloid cells, data visualisation and programming. -->

I am ready to educate the next generation of doctors and scientists and lead my own group to make transformative discoveries. 

My research aims to redefine our understanding of macrophages in inflammation. My holistic reserach view operates at the level of the gene, cell, tissue and organism. I have a particular interest in type 2 immune responses, single cell omics, lung & kidney immunology and autoimmunity. The concepts I uncovered in my recent publication in *Immunity* form the basis of my current grant applications where I will identify factors that control macrophage differentiation and function in inflammation.

# Current Roles 

##  Senior Research Fellow, Trinity College Dublin

-   I set up "TCD Omics" - single cell RNA-sequencing core service (independent project)
-   Myeloid cell dysfunction in COVID-19 (COVID-19 SFI-funded strategic partnership)
-   lab management and biomarker research (Trinity Kidney Health Centre with Mark Little).
-   Data integration of multi-modal clinical data, supervising development of R package (with Mark Little)
-   Longitudinal clinical study design (high parameter (30+) flow cytometry panels) - T cell exhaustion as a predictor of autoimmune relapse (PARADISE, with Mark Little)

## Honorary Research Fellow, University of Manchester

-   Active collaboration on *MRC* programme grant with Prof. Judith Allen (approx 25% FTE). 

## Employment History

```{r}
emp <- read_excel("../data/employment.xlsx") 
emp <- emp[-c(1:2),] # remove descriptive title
emp %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

## Education

```{r}
edu <- read_excel("../data/education.xlsx") 
edu <- edu[-c(1:2),] # remove descriptive title
cols <- colnames(edu)
edu  %>% arrange(desc(when)) %>%  detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

<!-- \newpage -->

# Funding

```{r}

gra <- read_excel("../data/grants.xlsx")
gra<- gra[-c(1:2),] # remove descriptive title
gra %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

## Funding, Under Review

<!-- -->

  **SFI Pathways program 2022**, â‚¬425,000.
    <!-- -   *Macrophage Activation and Transition in the Autoimmune Kidney (MacATAK): uncovering functional outcomes of macrophage differentiation in the inflamed kidney*. -->
    -   *Title withheld*.
    -   Short-listed for 2nd stage, expected outcome Q2 2023.

# Presentations, Awards & contributions

## Oral Presentations (details of 8X poster presentations availible in appendex)

```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what != 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```

## Awards

```{r}
awards <- read_excel("../data/awards.xlsx")
awards<- awards[-c(1:2),] # remove descriptive title
awards %>% arrange(desc(when))  %>% brief_entries(
              what = with ,# middle
              when = when  # left
    ) # right
```

## Discipline Contributions
```{r}
rev <- read_excel("../data/reviewing.xlsx")
rev<- rev[-c(1:2),] # remove descriptive title
rev  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,
              with = with
                # left
    ) # right
```


# Publications

```{r}
library(dplyr)
#knitr::write_bib(c("vitae", "tibble"), "My collection.bib")

bibliography_entries("My collection.bib") %>%
  arrange(desc(issued))

```

* Dwivedi, A., Ui Mhaonaigh, A., Carrol, M., Little, M and Finlay, C.M. low density neutrophils in SARS-Cov 2 **Under Review, I am corresponding author**

* [Commentry on my work in Immunity: Macrophages show up in style when Th2 lymphocytes organize their homecoming.](https://www.cell.com/immunity/fulltext/S1074-7613(23)00175-9)

*  [Paper featured on The Immunology Podcast](https://www.immunologypodcast.com/ep-51-the-journey-of-cells-featuring-dr-doug-green)

# Collaborations (active)

```{r}
#short version
  sup <- read_excel("../data/collaborations.xlsx")
sup<- sup[-c(1:2),] # remove descriptive title
sup  %>% arrange(desc(when)) %>% detailed_entries(
  #what = what, #subtitle
                         #when = when, # to right
                         what = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```
<!-- \newpage -->

```{r}
# #long versions  
# sup <- read_excel("../data/collaborations.xlsx")
# sup<- sup[-c(1:2),] # remove descriptive title
# sup  %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
#                          when = when, # to right
#                          why = why , # sub-subtitle
#                          with = with, # Major title
#                          where =  where) #Rightmost)

```


# Teaching & Supervision

<!-- ## Formal teaching -->
```{r}
#teaching short version 
# t <- read_excel("../data/teaching.xlsx")
# t<- t[-c(1:2),] # remove descriptive title
# t  %>% arrange(desc(when)) %>% brief_entries(
#               with= what ,# middle
#               when = when,
#               what = with# left
#     ) # right

t <- read_excel("../data/teaching.xlsx")



t %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

```{r}
# #teaching long version 
# t <- read_excel("../data/teaching.xlsx")
# t<- t[-c(1:2),] # remove descriptive title
# t  %>% arrange(desc(when)) %>% brief_entries(
#               with= what ,# middle
#               when = when,
#               what = with# left
#     ) # right

```

## Young scientist supervision
```{r}
# for longer version of CV
# sup <- read_excel("../data/supervison.xlsx")
# sup<- sup[-c(1:2),] # remove descriptive title
# sup  %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
#                          when = when, # to right
#                          why = why , # sub-subtitle
#                          with = with, # Major title
#                          where =  where) #Rightmost)
```

```{r}
sup <- read_excel("../data/supervison.xlsx")
sup<- sup[-c(1:2),] # remove descriptive title
sup$where <- sup$where %>% as.factor() %>% fct_recode("TCD"="Trinity College Dublin",
                                                                        "UoM" = "University of Manchester")

sup  %>% arrange(desc(when))  %>% brief_entries(
              what = what,# middle
              when = when, # left
              with = where # right
                # left
    ) # right
```

**Additional teaching duties** In 2016 I took on the academic duties for a Professor in TCD on teaching sabbatical. This included all aspects of the role, including practical supervision, assessment and marking, tutorials and administrative duties (including backboard). I have organised visits for school students at various points between 2013-2016. I have developed a 12 hour course in basic data visualisation skills for immunologists [basic data science skills for immunologist (click here)](https://conorisco.github.io/dataviz_immunology_r/).

I acted as official 'external mentor' of a PhD student in Wellcome Trust in Immunomatrix in Complex Disease PhD program in Manchester. I sit on the thesis committee of 3 Trinity PhD students in School of Medicine and Statistics. My teaching philosophy is Think-Pair-Share - a method to keep students engaged by creating a mini student led discussions in class.

# Outreach and organisation

**Discover Research Night**, Dublin Sep 2014. Marie SkÅ‚odowska-Curie-funded. I was the event organiser responsible for public engagement at Trinity Biomedical Sciences Institute, organising a team of 50 researchers in leading themed tours of scientific facilities. We were the most subscribed event of Dublin Research Night (\>500 members of the public) 8 weeks' full-time.

**10th International Symposium on Bordetella**, Sep 2013. Logistics manager responsible for operations of the conference including registration, AV equipment and support for speakers.

**Manchester Immunology Group Seminar Series**, May 2017 -- Sep 2018. Co-lead organiser. Handling invitations, hospitality, communications, for renowned international speakers to visit University of Manchester.

**School talk**, Oaklands Community College, Edenderry Co. Offaly, 2021. "The life of a scientist".

 **TCD Omics Core facility**. I formed a working group in 2021 to address the competitive disadvantage of the lack of single cell RNA-sequencing capacity in TCD. The service was launched in 2022 on a cost-for-service basis and delivers a full pipeline from consultation, cellular isolation, library preparation, sequencing, bioinformatics and basic downstream analysis. This has now received internal support for a 0.5 % FTE research assistant and we have offered services to 5 academic and 1 industry partners.
 
<!-- \newpage -->

# Referees

```{r}
ref <- read_excel("../data/referees.xlsx")
ref<- ref[-c(1:2),] # remove descriptive title
ref %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                        # why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

# Appendix

[Appendix on Github: **Media  & commentary**, **Manuscripts in preparation**, **Short-listed Applications**,**Poster Presentations**, **Thesis Committees and Vivas**, **Interview panels**, **Training**,**Courses** **Societies and Committees**, and **Skills**.](https://github.com/Conorisco/CV/raw/main/CV_conorisco/CV_conorisco.pdf) 


<!-- \newpage -->

\newpage

# Appendix

## Citations

```{r fig.height=2, fig.width=6}
p1<- ggplot(ct, aes(year, cites)) +
    geom_segment(aes(xend = year, yend = 0), size=1, color='darkgrey') +
  ylab('Citations') +
  labs(caption = "Citations per year. Source: Google Scholar") +
   annotate("text", x=2013, y=150, label= paste('h-index:', l$h_index)) +
     annotate("text", x=2013, y=120, label= paste('i10-index:', l$i10_index)) +
  scale_x_continuous(breaks = c(2010:2023)) +
  geom_point(size=3, color= col) +
  theme_cowplot(font_size = 10) +
  theme(axis.title.x = element_blank(),
        axis.text.x = element_text(angle = 45, hjust = 1),
         plot.caption = element_text(face = "italic"))

# p1
papers <- read_excel("../data/Publication details.xlsx")

mytheme <- gridExtra::ttheme_default(
    core = list(fg_params=list(cex = 0.5)),
    colhead = list(fg_params=list(cex = 0.6)),
    rowhead = list(fg_params=list(cex = 0.6)))

p2 <- tableGrob(papers,theme = mytheme,rows = NULL)


grid.arrange(p1, p2,
             nrow=1,
             as.table=TRUE)
```

## Manuscripts in Preparation or review, etc (pre-prints as listed as part of publications)

* Ridge, R.,  Finlay, C.M., Oâ€™ Farrelly, C., Moran, B., Alvarado-Vazquez, A., Hallgren, J., Conlon. CD34+ CD117+ FcÎµR1lo cells in peripheral blood are elevated in patients with chronic spontaneous urticaria and display phenotypes which correlate with treatment response

* Dwivedi, A., Ui Mhaonaigh, A., Carrol, M., Little, M and Finlay, C.M. Characterisation of low density neutrophils in SARS-Cov 2 patients (Under review)

* Finlay, C.M. Parkinson, J.P., Chan, B.H.K. and Allen, J.E. The proteome of the pleural fluid

* O'Conluain, R., Hollingsworth, s., Little, M. and Finlay, C.M., Soluble urokinase plasminogen activator receptor is a biomarker of kidney disease in ANCA-associated vasculitis

## Media and Commentary

* [Li, X., and Jakubzick, C. V. (2023). Macrophages show up in style when Th2 lymphocytes organize their homecoming. Immunity 56, 900â€“902. 10.1016/j.immuni.2023.04.007.](https://www.cell.com/immunity/fulltext/S1074-7613(23)00175-9)

*  [Immunity paper featured on The Immunology Podcast - Episode 51](https://www.immunologypodcast.com/ep-51-the-journey-of-cells-featuring-dr-doug-green)

* [https://www.tcd.ie/news_events/articles/2023/researchers-tackle-parasitic-worms-that-infect-millions-worldwide/]()

* Trinity College and University of Manchester tackle parasitic worms that infect millions, Irish Medical Times, Vol 74 No.4, April 2023

## Short-listed Applications

-   Short-listed for UKRI Innovation/Rutherford Fund Skills Development Fellowship (Computational Biology), 2017.

-   Short-listed for academic-industry partnership post-doctoral position, Belgium, 2019

-   Short-listed as candidate from school of medicine for Science Foundation Ireland Pathways programme, 2021

-   Short-listed for Lister Prize 2023

## Poster Presentations

```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what == 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```

## Training

```{r}
train <- read_excel("../data/training.xlsx")
#train<- train[-c(1:2),] # remove descriptive title
train  %>% arrange(desc(Year)) %>% brief_entries(
              what = Course,# middle
              when = Year,# left
              with = Insitution# right
    ) 
```

## Thesis Committees and Continuation Vivas

```{r}
tc <- read_excel("../data/Thesis_committee.xlsx")
tc<- tc[-c(1),] # remove descriptive title
tc  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,# left
              with = with# right
    ) 
```
## Interview Panels
```{r}
rev <- read_excel("../data/Interview panels.xlsx")
rev<- rev[-c(1:2),] # remove descriptive title
rev  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,
              with = with
                # left
    ) # right
```

## Courses

```{r}

cor <- read_excel("../data/courses.xlsx")

TCD <- cor$`Trinity College Dublin`
Manchester <- cor$`University of Manchester`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("University of Manchester",length(Manchester)),
          rep("Trinity College Dublin", length(TCD))), 
              detail = c(Manchester,TCD))
classes %>%
  detailed_entries(
    what = title,
    why = detail, .protect = F
  )
```

## Societies and Committees

```{r}

soc <- read_excel("../data/societies.xlsx")

socs <- soc$soc
com <- soc$`TCD Committees`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("Society Membership",length(socs)),
          rep("College Comittees", length(com))), 
              detail = c(socs,com))
classes %>%
  detailed_entries(
    what = title,
    why = detail#, .protect = F
  )
```

## Skills Annex

I have a broad skillet that includes animal research, translational research, molecular biology, bioinformatics and clinical data analysis that make me uniquely well suited to the design and management of research projects from a multidisciplinary point of view.

**Programming, Data analysis, Statistics & Visualisation**: R (Fluent, 6 years), Python (Intermediate, 4 years), RMarkdown (Intermediate, 4 years) Unix/Bash (Beginner). Data analysis/visualisation in and in R. Other programs: FlowJo and Prism, Adobe Illustrator, biostatistics and its application to experimental design and interpretation. Database creation and maintenance. R Shiny Apps. high-performance computing (basic).

**Bioinformatics** Expert in summarising take-home messages from single cell RNA-sequencing analysis in R and python (5 years experience). Mass cytometery analysis (FLowCore, Spectre). Analysis of microarray/bulk RNA-seq datasets analysis in R (limma, DESeq2), nCounter nanosting anlaysis. Pathway analysis (IPA, GO), regulatory network analysis (SCENIC), trajectory inference (Slingshot, RNA velocity), clustering (hierarchical and graph-based methods), dimension reduction (UMAP, Trimap, PCA), basic experience with machine learning (non linear, random forest) and systems biology (graphs, networks, logical modelling and differential equations).

**Animal models of Disease**: 12 years experience in *in vivo* model design: Autoimmunity (Experimental autoimmune encephalomyelitis, DSS-colitis, Imiquimod-induced inflammation), Cancer (B16 melanoma, CT26 lung), Infection (*L. sigmodontis*,*B. pertussis*, *F. hepatica*), Allergy (Allergen airway hypersensitivity) and general in vivo manipulation (Cell transfers, Irradiation/bone marrow transplantation peritonitis models, circadian rhythm modulation, cytokine administration, intraplueral injections).

**Laboratory techniques**: Expert in cell culture, cell sorting (BD Machines), high parameter (30+) flow cytometry, mass cytometry (including panel design, optimisation, storage, batch correction), tissue digestion, ELISA, lab management, inventory, ordering, health and safety.

**Molecular biology** NGS library preparation (Illumina) for single cell RNA-sequencing (BD Rhapsody and 10X genomics), Illumina sequecing (Novaaseq 6000) study design for NGS experiments, RNA extraction, RT-qPCR, nucleic acid QC (TapeStation, Qubit), PCR. Label free quantitative proteomics, bulk RNA-seq and ATAC-seq library preparation, enzyme assays.

**Clinical research and research data handling**: Ability to mange and interact with multidisciplinary teams of clinicians and scientist. Cell isolation from blood (Whole blood, granulocytes/neutrophils, PBMC), biobanking, clinical database management (RedCap).  Patient stratification, data curating and multi-modal integration: data analysis of longitudinal clinical datasets with 100s of clinical fields and 10000s of patients. Writing reports in RMarkdown to be read by clinicians and scientists.


```{r eval=FALSE}
tribble(
  ~Year, ~Type, ~Desc,
  1903, "Physics", "Awarded for her work on radioactivity with Pierre Curie and Henri Becquerel",
  1911, "Chemistry", "Awarded for the discovery of radium and polonium"
) %>% 
  brief_entries(
    glue::glue("Nobel Prize in {Type}"),
    Year, 
    Desc
  )
```

```{r eval=FALSE}
library(tibble)
tribble(
  ~ Degree, ~ Year, ~ Institution, ~ Where,
  "Informal studies", "1889-91", "Flying University", "Warsaw, Poland",
  "Master of Physics", "1893", "Sorbonne UniversitÃ©", "Paris, France",
  "Master of Mathematics", "1894", "Sorbonne UniversitÃ©", "Paris, France"
) %>% 
  detailed_entries(Degree, Year, Institution, Where)
```
