---
name: Conor M.
surname: Finlay
position: "Senior Research Fellow"
address: "Trinity Translational Medicines Insitute, Trinity College Dublin"
#phone: +353 857148825
profilepic: Conor.jpeg
#www: "github.com/Conorisco"
email: "cofinlay@tcd.ie; conor.finlay@manchester.ac.uk"
twitter: conorisco
github: conorisco
googlescholar: kQ6U7AkAAAAJ
#headcolor:
orcid: 0000-0001-8285-0903
#linkedin: mariecurie
date: "`r format(Sys.time(), '%B %Y')`"
output: vitae::awesomecv 
#output: vitae::twentyseconds
aboutme: "Immunologist with mutidiciplary skills in cellular, molecular, bioinformatics and clinical research"
#docname: "Conor Finlay CV.pdf"
#headcolor: 	050583
fontsize: 10pt # be aware thea LaTeX has limitations and cannot make text less than 10pt
headcolor: 224190
---

<!-- #\fontsize{11}{18}{8}{8} -->

```{r install packages, include=FALSE}
#install.packages("tidyverse")
#install.packages("vitae")
#install.packages("rorcid")
#install.packages('scholar')
#install.packages("readxl")
#install.packages("knitr")
#install.packages('tinytex')
#install.packages("gridExtra")
#tinytex::install_tinytex() # installs LaTeX - some issues on windows (but only some)
# knitting for first time will take a while as it appears to install a bunch of tools for latex on run 1
```

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
pacs <- c('vitae','tidyverse','readr','readxl','rorcid','rmarkdown','knitr','scholar','cowplot','gridExtra','patchwork')
lapply(pacs, library, character.only=TRUE) #%>% invisible()
myor <- "0000-0001-8285-0903"
myscol <- 'kQ6U7AkAAAAJ'
col <- "#050583"
```

```{r google scholar, include=FALSE}
l <- get_profile(myscol)
p <- get_publications(myscol)
ct <- get_citation_history(myscol)
```

<!-- # Research interests  -->

<!-- Cellular Immunologist with three core skills: *in vivo* **models of inflammation, bioinformatics and clinical research**. My main research interest is in what factors control mononuclear phagocyte (macrophage) differentiation during inflammation on a single cell level and how this impacts on disease outcomes. I have expertise T cell function, autoimmunity, type 2 immune responses, stromal-immune cell interactions, myeloid cells, data visualisation and programming. -->

# Snapshot

I am an energetic ambitious immunologist with a clear goal to build an independent research platform. My research interests include type 2 immune responses, autoimmunity and the implantation of computational approaches to analysis of immune cells and disease outcomes.

The ideas I generated from my recent work in *Immunity* forms the basis of 2 major grant applications under review where I aim to identify the factors that control mononuclear phagocyte (macrophage) differentiation during inflammation and how this impacts on disease outcomes.

I am ready to educate the next generation of doctors and scientists and lead my own group to make transformative discoveries in Immunology. I have a passion for enhancing research capacity through implementation of formalised structures to address the challenge of both generating 'omics' technology data and integrating this data with clinical databases.

# Current Roles

## Senior Research Fellow, Trinity College Dublin

Notable projects:

-   Myeloid cell dysfunction in COVID-19 (COVID-19 SFI-funded strategic partnership)
-   Core facility development in single cell RNA-sequencing ("TCD Omics", independent project)
-   lab management and biomarker research (Trinity Kidney Health Centre with Mark Little).
-   Data integration of multi-modal clinical data, supervising development of R package (with Mark Little)
-   Longitudinal clinical study design (high parameter (30+) flow cytometry panels) - T cell exhaustion as a predictor of autoimmune relapse (PARADISE, with Mark Little)
-   Bioinformatic support to multiple projects
-   Numerous active collaborations (see below)

## Honorary Research Fellow, University of Manchester

-   Collaboration on *MRC* programme grant with Prof. Judith Allen.
-   Ongoing collaborations with Doug Dyer and Mark Travis groups

# Employment History

```{r}
emp <- read_excel("../data/employment.xlsx") 
emp <- emp[-c(1:2),] # remove descriptive title
emp %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

# Education

```{r}
edu <- read_excel("../data/education.xlsx") 
edu <- edu[-c(1:2),] # remove descriptive title
cols <- colnames(edu)
edu  %>% arrange(desc(when)) %>%  detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

<!-- \newpage -->

# Funding

```{r}

gra <- read_excel("../data/grants.xlsx")
gra<- gra[-c(1:2),] # remove descriptive title
gra %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

# Funding (under review)

1.  **The Lister Institute of Preventive Medicine Prize 2023**, £250,000. Under international review
    -   *Title withheld*.
    -   Short-listed for 2nd stage, Expected outcome May 2023.

<!-- -->

2.  **SFI Pathways program 2022**, €425,000.
    <!-- -   *Macrophage Activation and Transition in the Autoimmune Kidney (MacATAK): uncovering functional outcomes of macrophage differentiation in the inflamed kidney*. -->
    -   *Title withheld*.
    -   Short-listed for 2nd stage, expected outcome April 2023.


# Presentations & awards

## Oral

```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what != 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```
\newpage
## Poster

```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what == 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```

## Awards

```{r}
awards <- read_excel("../data/awards.xlsx")
awards<- awards[-c(1:2),] # remove descriptive title
awards %>% arrange(desc(when))  %>% brief_entries(
              what = with ,# middle
              when = when  # left
    ) # right
```

# Publications

```{r fig.height=2, fig.width=6}
p1<- ggplot(ct, aes(year, cites)) +
    geom_segment(aes(xend = year, yend = 0), size=1, color='darkgrey') +
  ylab('Citations') +
  labs(caption = "Citations per year. Source: Google Scholar") +
   annotate("text", x=2013, y=150, label= paste('h-index:', l$h_index)) +
     annotate("text", x=2013, y=120, label= paste('i10-index:', l$i10_index)) +
  scale_x_continuous(breaks = c(2010:2023)) +
  geom_point(size=3, color= col) +
  theme_cowplot(font_size = 10) +
  theme(axis.title.x = element_blank(),
        axis.text.x = element_text(angle = 45, hjust = 1),
         plot.caption = element_text(face = "italic"))

# p1
papers <- read_excel("../data/Publication details.xlsx")

mytheme <- gridExtra::ttheme_default(
    core = list(fg_params=list(cex = 0.5)),
    colhead = list(fg_params=list(cex = 0.6)),
    rowhead = list(fg_params=list(cex = 0.6)))

p2 <- tableGrob(papers,theme = mytheme,rows = NULL)


grid.arrange(p1, p2,
             nrow=1,
             as.table=TRUE)
```

```{r}
library(dplyr)
#knitr::write_bib(c("vitae", "tibble"), "My collection.bib")

bibliography_entries("My collection.bib") %>%
  arrange(desc(issued))

```

# Collaborations (active)

```{r}
  sup <- read_excel("../data/collaborations.xlsx")
sup<- sup[-c(1:2),] # remove descriptive title
sup  %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

<!-- \newpage -->

# Teaching & Supervision

## Formal teaching

```{r}
#teaching short version 
# t <- read_excel("../data/teaching.xlsx")
# t<- t[-c(1:2),] # remove descriptive title
# t  %>% arrange(desc(when)) %>% brief_entries(
#               with= what ,# middle
#               when = when,
#               what = with# left
#     ) # right

t <- read_excel("../data/teaching.xlsx")



t %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

```{r}
# #teaching long version 
# t <- read_excel("../data/teaching.xlsx")
# t<- t[-c(1:2),] # remove descriptive title
# t  %>% arrange(desc(when)) %>% brief_entries(
#               with= what ,# middle
#               when = when,
#               what = with# left
#     ) # right

```

## Young scientist supervision

```{r}
sup <- read_excel("../data/supervison.xlsx")
sup<- sup[-c(1:2),] # remove descriptive title
sup  %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

**Additional teaching duties** In 2016 I took on the academic duties for a Professor in TCD on teaching sabbatical. This included all aspects of the role, including practical supervision, assessment and marking, tutorials and administrative duties (including backboard). I have organised visits for school students at various points between 2013-2016. I have developed a 12 hour course in basic data science skills for immunologists [basic data science skills for immunologist (click here)](https://conorisco.github.io/dataviz_immunology_r/). I acted as official 'external mentor' of a PhD student in Wellcome Trust in Immunomatrix in Complex Disease PhD program in Manchester. My teaching philosophy is Think-Pair-Share - a method to keep students engaged by creating a mini student led discussions in class.

# Outreach and organisation

**Discover Research Night**, Dublin Sep 2014. Marie Skłodowska-Curie-funded. I was the event organiser responsible for public engagement at Trinity Biomedical Sciences Institute, organising a team of 50 researchers in leading themed tours of scientific facilities. We were the most subscribed event of Dublin Research Night (\>500 members of the public) 8 weeks' full-time.

**10th International Symposium on Bordetella**, Sep 2013. Logistics manager responsible for operations of the conference including registration, AV equipment and support for speakers.

**Manchester Immunology Group Seminar Series**, May 2017 -- Sep 2018. Co-lead organiser. Handling invitations, hospitality, communications, for renowned international speakers to visit University of Manchester.

**School talk**, Oaklands Community College, Edenderry Co. Offaly, 2021. "The life of a scientist".


 **TCD Omics Core facility**. I formed a working group in 2021 to address the competitive disadvantage of not having a working single cell RNA-sequencing in TCD. The service was soft launched internally in 2022 on a cost-for-service basis and delivers a full pipeline from consultation, cellular isolation, library preparation, sequencing, bioinformatics and basic downstream analysis. I am to expand this to a full service and hire talent to expand the capability of this service to include external industry clients. 
<!-- \newpage -->

# Referees

```{r}
ref <- read_excel("../data/referees.xlsx")
ref<- ref[-c(1:2),] # remove descriptive title
ref %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                        # why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

<!-- \newpage -->

\newpage

# Appendix

## Skills

I have a broad skillet that includes animal research, translational research, molecular biology, bioinformatics and clinical data analysis that make me uniquely well suited to the design and management of research projects from a multi disciplinary point of view.

**Programming, Data analysis, Statistics & Visualisation**: R (Fluent, 6 years), Python (Intermediate, 4 years), RMarkdown (Intermediate, 4 years) Unix/Bash (Beginner). Data analysis/visualisation in and in R. Other programs: FlowJo and Prism, Adobe Illustrator, biostatistics and its application to experimental design and interpretation. Database creation and maintenance. R Shiny Apps. high-performance computing (basic).

**Bioinformatics** Expert in summarising take-home messages from single cell RNA-sequencing analysis in R and python (5 years experience). Mass cytometery analysis (FLowCore, Spectre). Analysis of microarray/bulk RNA-seq datasets analysis in R (limma, DESeq2), nCounter nanosting anlaysis. Pathway analysis (IPA, GO), regulatory network analysis (SCENIC), trajectory inference (Slingshot, RNA velocity), clustering (hierarchical and graph-based methods), dimension reduction (UMAP, Trimap, PCA), basic experience with machine learning (non linear, random forest) and systems biology (graphs, networks, logical modelling and differential equations).

**Animal models of Disease**: 12 years experience in *in vivo* model design: Autoimmunity (Experimental autoimmune encephalomyelitis, DSS-colitis, Imiquimod-induced inflammation), Cancer (B16 melanoma, CT26 lung), Infection (*L. sigmodontis*,*B. pertussis*, *F. hepatica*), Allergy (Allergen airway hypersensitivity) and general in vivo manipulation (Cell transfers, Irradiation/bone marrow transplantation peritonitis models, circadian rhythm modulation, cytokine administration, intraplueral injections).

**Laboratory techniques**: Expert in cell culture, cell sorting (BD Machines), high parameter (30+) flow cytometry, mass cytometry (including panel design, optimisation, storage, batch correction), tissue digestion, ELISA, lab management, inventory, ordering, health and safety.

**Molecular biology** NGS library preparation (Illumina) for single cell RNA-sequencing (BD Rhapsody and 10X genomics), Illumina sequecing (Novaaseq 6000) study design for NGS experiments, RNA extraction, RT-qPCR, nucleic acid QC (TapeStation, Qubit), PCR. Label free quantitative proteomics, bulk RNA-seq and ATAC-seq library preparation, enzyme assays.

**Clinical research and research data handling**: Ability to mange and interact with multi-disciplinary teams of clinicians and scientist. Cell isolation from blood (Whole blood, granulocytes/neutrophils, PBMC), biobanking, clinical database management (RedCap).  Patient stratification, data curating and multi-modal integration: data analysis of longitudinal clinical datasets with 100s of clinical fields and 10000s of patients. Writing reports in RMarkdown to be read by clinicians and scientists.

## Discipline and College contributions

```{r}
rev <- read_excel("../data/reviewing.xlsx")
rev<- rev[-c(1:2),] # remove descriptive title
rev  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,
              with = with
                # left
    ) # right

```

## Training

```{r}
train <- read_excel("../data/training.xlsx")
#train<- train[-c(1:2),] # remove descriptive title
train  %>% arrange(desc(Year)) %>% brief_entries(
              what = Course,# middle
              when = Year,# left
              with = Insitution# right
                
    ) 

```

## Courses

```{r}

cor <- read_excel("../data/courses.xlsx")


TCD <- cor$`Trinity College Dublin`
Manchester <- cor$`University of Manchester`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("University of Manchester",length(Manchester)),
          rep("Trinity College Dublin", length(TCD))), 
              detail = c(Manchester,TCD))
classes %>%
  detailed_entries(
    what = title,
    why = detail, .protect = F
  )
```

## Societies and committees

```{r}

soc <- read_excel("../data/societies.xlsx")

socs <- soc$soc
com <- soc$`TCD Committees`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("Society Membership",length(socs)),
          rep("College Comittees", length(com))), 
              detail = c(socs,com))
classes %>%
  detailed_entries(
    what = title,
    why = detail#, .protect = F
  )
```

<!-- ## Shortlisted applications -->

<!-- -   Shortlisted for UKRI Innovation/Rutherford Fund Skills Development Fellowship (Computational Biology), 2017. -->

<!-- -   Shortlisted as candidate from school of medicine for Science Foundation Ireland Pathways programme, 2021 -->

<!-- -   Shortlisted for academic-industry partnership with Janssen Pharmaceuticals, Belgium, 2019 -->

## Manuscripts in preparation

Dwivedi, A., Ui Mhaonaigh, A., Carrol, M., Little, M and Finlay, C.M. Characterisation of low density neutrophils in SARS-Cov 2 patients

Finlay, C.M. Parkinson, J.P., Chan, B.H.K. and Allen, J.E. The proteome of the pleural fluid

O'Conluain, R., Hollingsworth, s., Little, M. and Finlay, C.M., Soluble urokinase plasminogen activator receptor is a biomarker of kidney disease in ANCA-associated vasculitis

```{r eval=FALSE}
tribble(
  ~Year, ~Type, ~Desc,
  1903, "Physics", "Awarded for her work on radioactivity with Pierre Curie and Henri Becquerel",
  1911, "Chemistry", "Awarded for the discovery of radium and polonium"
) %>% 
  brief_entries(
    glue::glue("Nobel Prize in {Type}"),
    Year, 
    Desc
  )
```

```{r eval=FALSE}
library(tibble)
tribble(
  ~ Degree, ~ Year, ~ Institution, ~ Where,
  "Informal studies", "1889-91", "Flying University", "Warsaw, Poland",
  "Master of Physics", "1893", "Sorbonne Université", "Paris, France",
  "Master of Mathematics", "1894", "Sorbonne Université", "Paris, France"
) %>% 
  detailed_entries(Degree, Year, Institution, Where)
```
