---
name: Conor M.
surname: Finlay
position: "Senior Research Fellow"
address: "Trinity Translational Medicines Insitute, Trinity College Dublin"
#phone: +353 857148825
profilepic: Conor.jpeg
#www: "github.com/Conorisco"
email: "cofinlay@tcd.ie; conor.finlay@manchester.ac.uk"
twitter: conorisco
github: conorisco
googlescholar: kQ6U7AkAAAAJ
#headcolor:
orcid: 0000-0001-8285-0903
#linkedin: mariecurie
date: "`r format(Sys.time(), '%B %Y')`"
output: vitae::awesomecv 
#output: vitae::twentyseconds
aboutme: "Cellular immunologist with bioinformatic skills"
#docname: "Conor Finlay CV.pdf"
#headcolor: 414141
---

```{r install packages, include=FALSE}
#install.packages("tidyverse")
#install.packages("vitae")
#install.packages("rorcid")
#install.packages('scholar')
#install.packages("readxl")
#install.packages("knitr")
#install.packages('tinytex')
#install.packages("gridExtra")
#tinytex::install_tinytex() # installs LaTeX - some issues on windows (but only some)
# knitting for first time will take a while as it appears to install a bunch of tools for latex on run 1
```


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
pacs <- c('vitae','tidyverse','readr','readxl','rorcid','rmarkdown','knitr','scholar','cowplot','gridExtra')
lapply(pacs, library, character.only=TRUE) #%>% invisible()
myor <- "0000-0001-8285-0903"
myscol <- 'kQ6U7AkAAAAJ'
```

```{r google scholar, include=FALSE}
l <- get_profile(myscol)
p <- get_publications(myscol)
ct <- get_citation_history(myscol)

```


# Research interests 
I am a cellular Immunologist with three core skills: *in vivo* models of inflammation, bioinformatics and clinical research. My current research obsession is the regulation of mononuclear phagocyte differentiation during inflammation on a single cell level. 

# Current Roles
## Senior Research Fellow, Trinity College Dublin
 * COVID19 Science Foundation Ireland-funded strategic partnership
 * TCD Single Cell Core development
 * Trinity Kidney Health Centre, Prof. Mark A Little.

## Honorary Research Fellow, University of Manchester
 * Co-managing Medical Research Council programme grant with Prof Allen
 * Bioinformatics collaborations 
 * Core facility engagement
 
# Past Employment
```{r}
emp <- read_excel("../data/employment.xlsx") 
emp <- emp[-c(1:2),] # remove descriptive title
emp %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```
\newpage
# Education
```{r}
edu <- read_excel("../data/education.xlsx") 
edu <- edu[-c(1:2),] # remove descriptive title
cols <- colnames(edu)
edu  %>% arrange(desc(when)) %>%  detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```
# Skills

 **Animal models of Disease**: Experimental autoimmune encephalomyelitis, Litomosoides sigmodontis infection, OVA/Alum and dust mite-induced airway hypersensitivity, dextran sodium sulphate-induced colitis, circadian rhythm modulation, B16 melanoma, Sterile peritonitis, IL-4-complex administration, House airway hypersensitivity, Fasciola hepatica infection, Bordetella pertussis infection.

**Laboratory techniques**: In vitro cell culture, FACS & MACS cell sorting, flow cytometry (expert), mass cytometry, tissue digestion, ELISA, enzyme assays, RT-qPCR, confocal microscopy, Amnis Image Stream, sample preparation for mass spectrometry and single cell RNA-sequencing. Basic experience with PCR, genotyping, immunoblots and library preparation for Next Generation Sequencing.

**Clinical research**: Preparation of cells from human blood. Biobanking, clinical database management. Patient stratification and statistical analysis. Working in as part of a multi-disciplinary scientific-clinical research team. Data curating and integration: data analysis of longitudinal clinical datasets with 100s of clincal fields and 10000s of patients. Writing reports in RMarkdown to be read by clinicians and scientists. 

**Programming, Data analysis, Statistics & Visualisation**: R (Fluent), Python (Intermediate), RMarkdown (Intermediate) Unix/Bash (Beginner). Data analysis/visualisation in and in R using Tidyverse and ggplot2. Advanced user of Microsoft Office applications, TreeStar FlowJo and GraphPad Prism. Competent user of Adobe Illustrator and pathway analysis tools e.g. IPA. Strong background in statistics and its application to experimental design and interpretation of results. Database creation and maintenance. Creation of Shiny Apps. Basic knowledge of computer science and use of high-performance computer clusters. 

**Bioinformatics**. Analysis of microarray/bulk RNA-seq datasets analysis using the DESeq2 and EdgeR packages. Single cell analysis using Scater within SingleCellExperiment, pySCENIC, Velocyto, Seurat, FlowCore, FlowSOM, CytofKit. Dimensionality reduction: PCA, t-SNE and UMAP. Progression analysis using monocle3, Destiny and RNA velocity. Clustering using Phonograph, DevSM, FlowSOM hclust and treeclust.

# Funding
```{r}

gra <- read_excel("../data/grants.xlsx")
gra<- gra[-c(1:2),] # remove descriptive title
gra %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

*	Shortlisted for UKRI Innovation/Rutherford Fund Skills Development Fellowship (Computational Biology), Sep 2017.


\newpage

# Publications
```{r}
library(dplyr)
#knitr::write_bib(c("vitae", "tibble"), "My collection.bib")

bibliography_entries("My collection.bib") %>%
  arrange(desc(issued))

```
<!-- Total citations: `r l$total_cites`, h index: `r l$h_index`  -->
```{r fig.height=1.4, fig.width=2.4}
ggplot(ct, aes(year, cites)) +
    geom_segment(aes(xend = year, yend = 0), size=1, color='darkgrey') +
  ylab('Citations') +
   annotate("text", x=2011.5, y=150, label= paste('h-index:', l$h_index)) +
  geom_point(size=3, color='red') +
  theme_cowplot(font_size = 10) +
  theme(axis.title.x = element_blank())
```


# Presentations
## Oral
```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what != 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```
## Poster
```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what == 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```

# Awards
```{r}
awards <- read_excel("../data/awards.xlsx")
awards<- awards[-c(1:2),] # remove descriptive title
awards %>% arrange(desc(when))  %>% brief_entries(
              what = with ,# middle
              when = when  # left
    ) # right

```
# Outreach and organisation

* **Discover Research Night**, Dublin Sep 2014. Marie Skłodowska-Curie-funded. I was the event organiser responsible for public engagement at Trinity Biomedical Sciences Institute, 8 weeks’ full-time. 

* **10th International Symposium on Bordetella**, Sep 2013. Logistics manager responsible for operations of the conference including registration, AV equipment and support for speakers.

* **Manchester Immunology Group Seminar Series**, May 2017 – Sep 2018. Co-lead organiser. Handling invitations, hospitality, communications, for renowned international speakers to visit University of Manchester.

* **School talk**, Oaklands Community College, Edenderry Co. Offaly, 2021. “The life of a scientist”. 
* **Single Cell TCD Core facility**. I formed a working group in 2021 in Trinity college dublin to address the competitive disadvantage of not having a working single cell RNA-sequencing. My aim is to deliver a single cell RNA-seq service to TCD researchers on a cost per service basis that will act as a platform to develop a full core facility service.


# Teaching
```{r}
t <- read_excel("../data/teaching.xlsx")
t<- t[-c(1:2),] # remove descriptive title
t  %>% arrange(desc(when)) %>% brief_entries(
              with= what ,# middle
              when = when,
              what = with# left
    ) # right

```

Other duties include Undergraduate practical supervision, tutorials, and organising visits of school students as various points between 2013-2016. [Introduction to data scientists for immunologists course available here](https://conorisco.github.io/dataviz_immunology_r/  ). Acting as an official ‘external mentor’ of a PhD student in Wellcome Trust in Immunomatrix in Complex Disease PhD program in Manchester.

# Supervision
```{r}
sup <- read_excel("../data/supervison.xlsx")
sup<- sup[-c(1:2),] # remove descriptive title
sup  %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```
# Training
## Formal
```{r}
train <- read_excel("../data/training.xlsx")
train<- train[-c(1:2),] # remove descriptive title
train %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

## Courses
```{r}

cor <- read_excel("../data/courses.xlsx")


TCD <- cor$`Trinity College Dublin`
Manchester <- cor$`University of Manchester`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("University of MAnchester",length(Manchester)),
          rep("Trinity College Dublin", length(TCD))), 
              detail = c(Manchester,TCD))
classes %>%
  detailed_entries(
    what = title,
    why = detail, .protect = F
  )
```
# Other Contributions 

## Reviewing
```{r}
rev <- read_excel("../data/reviewing.xlsx")
rev<- rev[-c(1:2),] # remove descriptive title
rev  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,
              with = with
                # left
    ) # right

```

## Societies and committees
```{r}

soc <- read_excel("../data/societies.xlsx")

socs <- soc$soc
com <- soc$`TCD Committees`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("Society Membership",length(socs)),
          rep("Internall Comittees", length(com))), 
              detail = c(socs,com))
classes %>%
  detailed_entries(
    what = title,
    why = detail#, .protect = F
  )
```

# Referees
```{r}
ref <- read_excel("../data/referees.xlsx")
ref<- ref[-c(1:2),] # remove descriptive title
ref %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                        # why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```


```{r eval=FALSE}
tribble(
  ~Year, ~Type, ~Desc,
  1903, "Physics", "Awarded for her work on radioactivity with Pierre Curie and Henri Becquerel",
  1911, "Chemistry", "Awarded for the discovery of radium and polonium"
) %>% 
  brief_entries(
    glue::glue("Nobel Prize in {Type}"),
    Year, 
    Desc
  )
```

```{r eval=FALSE}
library(tibble)
tribble(
  ~ Degree, ~ Year, ~ Institution, ~ Where,
  "Informal studies", "1889-91", "Flying University", "Warsaw, Poland",
  "Master of Physics", "1893", "Sorbonne Université", "Paris, France",
  "Master of Mathematics", "1894", "Sorbonne Université", "Paris, France"
) %>% 
  detailed_entries(Degree, Year, Institution, Where)
```



