---
name: Conor M.
surname: Finlay
position: "Senior Research Fellow"
address: "Trinity Translational Medicines Insitute, Trinity College Dublin"
#phone: +353 857148825
profilepic: Conor.jpeg
#www: "github.com/Conorisco"
email: "cofinlay@tcd.ie; conor.finlay@manchester.ac.uk"
twitter: conorisco
github: conorisco
googlescholar: kQ6U7AkAAAAJ
#headcolor:
orcid: 0000-0001-8285-0903
#linkedin: mariecurie
date: "`r format(Sys.time(), '%B %Y')`"
output: vitae::awesomecv 
#output: vitae::twentyseconds
aboutme: "Cellular Immunologist with bioinformatics experience"
#docname: "Conor Finlay CV.pdf"
headcolor: 	050583
fontsize: 10pt
---
<!-- #\fontsize{11}{18}{8}{8} -->
```{r install packages, include=FALSE}
#install.packages("tidyverse")
#install.packages("vitae")
#install.packages("rorcid")
#install.packages('scholar')
#install.packages("readxl")
#install.packages("knitr")
#install.packages('tinytex')
#install.packages("gridExtra")
#tinytex::install_tinytex() # installs LaTeX - some issues on windows (but only some)
# knitting for first time will take a while as it appears to install a bunch of tools for latex on run 1
```


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
pacs <- c('vitae','tidyverse','readr','readxl','rorcid','rmarkdown','knitr','scholar','cowplot','gridExtra')
lapply(pacs, library, character.only=TRUE) #%>% invisible()
myor <- "0000-0001-8285-0903"
myscol <- 'kQ6U7AkAAAAJ'
col <- "#050583"
```

```{r google scholar, include=FALSE}
l <- get_profile(myscol)
p <- get_publications(myscol)
ct <- get_citation_history(myscol)

```


# Research interests 
Cellular Immunologist with three core skills: *in vivo* **models of inflammation, bioinformatics and clinical research**. My main research interest is in what factors control mononuclear phagocyte (macrophage) differentiation during inflammation on a single cell level and how this impacts on disease outcomes. I have expertise T cell function, autoimmunity, type 2 immune responses, stromal-immune cell interactions, myeloid cells, data visualisation and programming.

# Current Roles
## Senior Research Fellow, Trinity College Dublin
 * COVID19 Science Foundation Ireland-funded strategic partnership.
 * Chairing the development of a new core facility service in single cell RNA-sequencing ("TCD Single Cell")
 * Trinity Kidney Health Centre, lab management and clinical research.

## Honorary Research Fellow, University of Manchester
 * Co-managing *MRC* programme grant with Prof. Judith Allen includes bi-monthly visits and weekly online meetings
 * Collaborations with Judith Allen, Mark Travis and Matthew Hepworth groups (providing bioinformatics analysis)

# Employment History
```{r}
emp <- read_excel("../data/employment.xlsx") 
emp <- emp[-c(1:2),] # remove descriptive title
emp %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

# Education
```{r}
edu <- read_excel("../data/education.xlsx") 
edu <- edu[-c(1:2),] # remove descriptive title
cols <- colnames(edu)
edu  %>% arrange(desc(when)) %>%  detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

# Publications
```{r}
library(dplyr)
#knitr::write_bib(c("vitae", "tibble"), "My collection.bib")

bibliography_entries("My collection.bib") %>%
  arrange(desc(issued))

```

## Unpublished manuscipts in preparation

Finlay, C.M. Parkinson, J.P., Chan, B.H.K. and Allen, J.E. Contribution of tissue resident macrophages to the proteome of the pleural fluid (running title)

O'Conluain, R., Hollingsworth, s., Little, M. and Finlay, C.M.,  Soluble urokinase plasminogen activator receptor is a biomarker of kidney disease in ANCA-associated vasculitis (running title)

Dwivedi, A., Ui Mhaonaigh, A.,  Carrol, M. Finlay, C.M. and Little, M. Characterisation of low density neutrophils in SARS-Cov 2 patients (running title)

\newpage
# Funding
```{r}

gra <- read_excel("../data/grants.xlsx")
gra<- gra[-c(1:2),] # remove descriptive title
gra %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

# Skills

 **Animal models of Disease**: 12 years experience in in vivo model design: Autoimmunity (Experimental autoimmune encephalomyelitis, DSS-colitis, Imiquimod-induced inflammation), Cancer (B16 melanoma, CT26 lung), Infection (*L. sigmodontis*,*B. pertussis*, *F. hepatica*), Allergy (Allergen airway hypersensitivity) and general in vivo manipulation (Cell transfers, Irradiation/bone marrow transplantation peritonitis models, circadian rhythm modulation, cytokine administration, intraplueral injections).


**Laboratory techniques**: Expert in cell culture, cell sorting (BD Machines), high parameter (20+) flow cytometry, mass cytometry (including panel design), tissue digestion, ELISA, lab management, inventory, ordering, health and saftey. 

**Molecular biology** NGS library preparation (Illumina) for single cell RNA-sequencing (BD Rhapsody and 10X genomics), study design for NGS experiments, RNA extraction, RT-qPCR, nucleic acid QC (TapeStation, Qubit), PCR. Label free quanitiative proteomics, bulk RNA-seq and ATAC-seq library preparation, enzyme assays. 

**Clinical research**: Cell isolation from blood (Whole blood, granulocytes/neutrophils, PBMC), biobanking, clinical database management (RedCap). Patient stratification, data curating and multi-modal integration: data analysis of longitudinal clinical datasets with 100s of clinical fields and 10000s of patients. Writing reports in RMarkdown to be read by clinicians and scientists. 

**Programming, Data analysis, Statistics & Visualisation**: R (Fluent, 5 years), Python (Intermediate, 3 years), RMarkdown (Intermediate, 3 years) Unix/Bash (Beginner). Data analysis/visualisation in and in R. Other programs: FlowJo and Prism, Adobe Illustrator, biostatistics and its application to experimental design and interpretation. Database creation and maintenance. R Shiny Apps. high-performance computing (basic). 

**Bioinformatics** Expert in summarising take-home messages from single cell RNA-sequencing analysis in R and python (4 years experience). Mass cytometery analysis (FLowCore, Spectre). Analysis of microarray/bulk RNA-seq datasets analysis in R (limma, DESeq2). Pathway analysis (IPA, GO), regulatory network analysis (SCENIC), trajectory inference (Slingshot, RNA velocity), clustering (hierarchical and graph-based methods), dimension reduction (UMAP, Trimap, PCA, PAC-map), basic experience with machine learning (non linear, random forest) and systems biology (graphs, networks, logical modelling and differential equations). 

# Presentations & awards
## Oral
```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what != 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```
## Poster
```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what == 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```
## Awards
```{r}
awards <- read_excel("../data/awards.xlsx")
awards<- awards[-c(1:2),] # remove descriptive title
awards %>% arrange(desc(when))  %>% brief_entries(
              what = with ,# middle
              when = when  # left
    ) # right
```
<!-- \newpage -->
# Teaching & Supervison
## Lecturing
```{r}
t <- read_excel("../data/teaching.xlsx")
t<- t[-c(1:2),] # remove descriptive title
t  %>% arrange(desc(when)) %>% brief_entries(
              with= what ,# middle
              when = when,
              what = with# left
    ) # right

```

**Other teaching related duties** Undergraduate practical supervision, tutorials, and organising visits of school students as various points between 2013-2016. [Introduction to data scientists for immunologists course (click here)](https://conorisco.github.io/dataviz_immunology_r/  ). Official ‘external mentor’ of a PhD student in Wellcome Trust in Immunomatrix in Complex Disease PhD program in Manchester. Assessor for the PhD continuation viva (2022). Interview panel for Post-doctoral researcher (2022) and research assistant (2021). 

## Supervision
```{r}
sup <- read_excel("../data/supervison.xlsx")
sup<- sup[-c(1:2),] # remove descriptive title
sup  %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

# Outreach and organisation

**Discover Research Night**, Dublin Sep 2014. Marie Skłodowska-Curie-funded. I was the event organiser responsible for public engagement at Trinity Biomedical Sciences Institute, 8 weeks’ full-time. 

**10th International Symposium on Bordetella**, Sep 2013. Logistics manager responsible for operations of the conference including registration, AV equipment and support for speakers.

**Manchester Immunology Group Seminar Series**, May 2017 – Sep 2018. Co-lead organiser. Handling invitations, hospitality, communications, for renowned international speakers to visit University of Manchester.

**School talk**, Oaklands Community College, Edenderry Co. Offaly, 2021. “The life of a scientist”. 

**Single Cell TCD Core facility**. I formed a working group in 2021 to address the competitive disadvantage of not having a working single cell RNA-sequencing in TCD. The service was soft launched internally in 2022 on a cost-for-service basis and delivers a full pipeline from consultation, cellular isolation, library preparation, sequencing, bioinformatics and basic downstream analysis. I am to expand this to a full service and hire talent to expand the capability of this sercive to include external industry clinets.

<!-- \newpage -->

# Referees
```{r}
ref <- read_excel("../data/referees.xlsx")
ref<- ref[-c(1:2),] # remove descriptive title
ref %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                        # why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

\newpage

# Appendix
## Shortlisted applications
*	Shortlisted for UKRI Innovation/Rutherford Fund Skills Development Fellowship (Computational Biology), 2017.
* Shortlisted as candidate from school of medicine for Science Foundation Ireland Pathways programme, 2021
* Shortlisted for academic-industry partnership with Janssen Pharmaceuticals, Belgium, 2019

## Reviewing
```{r}
rev <- read_excel("../data/reviewing.xlsx")
rev<- rev[-c(1:2),] # remove descriptive title
rev  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,
              with = with
                # left
    ) # right

```

## Training

```{r}
train <- read_excel("../data/training.xlsx")
#train<- train[-c(1:2),] # remove descriptive title
train  %>% arrange(desc(Year)) %>% brief_entries(
              what = Course,# middle
              when = Year,# left
              with = Insitution# right
                
    ) 

```

## Courses
```{r}

cor <- read_excel("../data/courses.xlsx")


TCD <- cor$`Trinity College Dublin`
Manchester <- cor$`University of Manchester`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("University of Manchester",length(Manchester)),
          rep("Trinity College Dublin", length(TCD))), 
              detail = c(Manchester,TCD))
classes %>%
  detailed_entries(
    what = title,
    why = detail, .protect = F
  )
```

## Societies and committees
```{r}

soc <- read_excel("../data/societies.xlsx")

socs <- soc$soc
com <- soc$`TCD Committees`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("Society Membership",length(socs)),
          rep("College Comittees", length(com))), 
              detail = c(socs,com))
classes %>%
  detailed_entries(
    what = title,
    why = detail#, .protect = F
  )
```

## Citation metrics
<!-- Total citations: `r l$total_cites`, h index: `r l$h_index`  -->

```{r fig.height=2, fig.width=2.5}
ggplot(ct, aes(year, cites)) +
    geom_segment(aes(xend = year, yend = 0), size=1, color='darkgrey') +
  ylab('Citations') +
  labs(caption = "Citations per year. Source: Google Scholar") +
   annotate("text", x=2013, y=150, label= paste('h-index:', l$h_index)) +
  scale_x_continuous(breaks = c(2010:2022)) +
  geom_point(size=3, color= col) +
  theme_cowplot(font_size = 10) +
  theme(axis.title.x = element_blank(),
        axis.text.x = element_text(angle = 45, hjust = 1),
         plot.caption = element_text(face = "italic"))
```


```{r eval=FALSE}
tribble(
  ~Year, ~Type, ~Desc,
  1903, "Physics", "Awarded for her work on radioactivity with Pierre Curie and Henri Becquerel",
  1911, "Chemistry", "Awarded for the discovery of radium and polonium"
) %>% 
  brief_entries(
    glue::glue("Nobel Prize in {Type}"),
    Year, 
    Desc
  )
```

```{r eval=FALSE}
library(tibble)
tribble(
  ~ Degree, ~ Year, ~ Institution, ~ Where,
  "Informal studies", "1889-91", "Flying University", "Warsaw, Poland",
  "Master of Physics", "1893", "Sorbonne Université", "Paris, France",
  "Master of Mathematics", "1894", "Sorbonne Université", "Paris, France"
) %>% 
  detailed_entries(Degree, Year, Institution, Where)
```



