---
name: Conor M.
surname: Finlay
position: "Research Assistant Professor"
address: "Trinity Translational Medicine Institute, Trinity College Dublin"
phone: +353 857148825
profilepic: Conor.jpeg
#www: "github.com/Conorisco"
email: "cofinlay@tcd.ie; conor.finlay@manchester.ac.uk"
twitter: conorisco
github: conorisco
googlescholar: kQ6U7AkAAAAJ
#headcolor:
orcid: 0000-0001-8285-0903
date: "`r format(Sys.time(), '%B %Y')`"
output: vitae::awesomecv
#output: vitae::twentyseconds
aboutme: "Immunologist"
#docname: "Conor Finlay CV.pdf"
#headcolor: 	050583
fontsize: 10pt # be aware thea LaTeX has limitations and cannot make text less than 10pt
headcolor: 224190
header-includes:
  - \providecommand{\pandocbounded}[1]{#1}
---

<!-- # Career profile -->
<!-- I started my career in Trinity College Dublin identify pathways that shut down autoimmunity and allergy, leading to three papers in *The Journal of Immunology* and one in *Parasite immunology* and co-authorship in *Nature communications*. Next in Manchester I made a major advance on how adaptive immunity controls macrophages and how macrophage differentiation controls disease outcomes, publishing articles in *Immunity*, *elife*, *Nature Immunology* and *Parasite Immunology*. Recently, back in Ireland I have been conducting patient orientated research where I have submitted my first co-final author paper. I have received a best talk at the Irish Society of Immunology conference, been an invited speaker at 3 conferences and spoken at 13 more. I have received 8 awards. I have supervised 13 research students, reviewed articles, grants and will soon join *Frontiers in immunology* as a review editor. I am the member of 4 scientific societies, and sit on 3 committees. I am a co-inventor on a patent WM01-740-02 that has been commercialised as a biotech company Pravalix Tx.  -->

<!-- I am ready to educate the next generation of scientists and clinicians and lead my own group to make transformative discoveries. -->

<!-- #\fontsize{11}{18}{8}{8} -->
```{r install packages, include=FALSE}
#install.packages("tidyverse")
#install.packages("vitae")
#install.packages("rorcid")
#install.packages('scholar')
#install.packages("readxl")
#install.packages("knitr")
#install.packages('tinytex')
#install.packages("gridExtra")
#tinytex::install_tinytex() # installs LaTeX - some issues on windows (but only some)
# knitting for first time will take a while as it appears to install a bunch of tools for latex on run 1
```

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
pacs <- c('vitae','tidyverse','readr','readxl','rorcid','rmarkdown','knitr','scholar','cowplot','gridExtra','patchwork')
lapply(pacs, library, character.only=TRUE) #%>% invisible()
myor <- "0000-0001-8285-0903"
myscol <- 'kQ6U7AkAAAAJ'
col <- "#050583"
```

```{r google scholar, include=FALSE}
l <- get_profile(myscol)
p <- get_publications(myscol)
ct <- get_citation_history(myscol)
```

<!-- # Snapshot -->
<!-- I am principle investigator of a research group that focuses on how we can  -->

<!--  mononuclear phagocyte devlopment during inflammation on a single cell level and how this impacts on disease outcomes. I have expertise in single cell RNA-sequencing, NGS, cell biology, *in vitro* and *in vivo* sciences and summarising complex data through effective data visualisation. -->
<!-- # SECTION 1 - applicant details -->

# Research Roles

<!-- -   Principal Investigator Finlay laboratory (Immune cell biology and bioinformatics, 2 PhD students), 2024-present -->
<!-- -   Founder and scientific director of Trinity Omics - Single cell RNA-sequencing core facility, 2021-present  -->
<!-- -   Trinity Kidney Centre, immunophenotyping and biomarker research, 2022-present -->
<!-- -   Pleural disease with Associate Prof. Luara Gleeson, 2023-present -->
<!-- -   Mast cell biology, collaboration with Katie Ridge. 2022-present -->
## Research Assistant Professor, Trinity College Dublin, 2025-present
My group investigates innate immune cells in inflammation, combining molecular immunology, in vivo science and computational approaches to understand immune cell function in health and disease. 

```{r}
library(knitr)
library(kableExtra)

data <- data.frame(
  Role = c("PI, Finlay Lab", "Acedemic Director, Trinity Omics", "Renal Research", "Pleural Disease Program"),
  #Period = c("2024-present", "2021-present", "2022-present", "2023-present", "2022-present"),
  Focus = c("Immune cell biology & bioinformatics", "Single cell RNA-seq facility", "Biomarker discovery", "Macrophage function"),
  Partners = c("3 PhD students", "TTMI-supported RA", "Prof. Mark Little, ADAPT", "Assoc. Prof. Laura Gleeson"),
  Outputs = c("Macrophage biology, computational immunology", "Core technology service", "Data analytics, translational research", "Translational immunology")
)

# kable(data, format = "latex", booktabs = TRUE,
#       col.names = c("Role", "Focus", "Team/Partners", "Key Outputs"))
# 
# kable(data, format = "latex", booktabs = TRUE) %>%
#   kable_styling(latex_options = c("striped", "hold_position"))
# 
# kable(data, format = "latex", booktabs = TRUE) %>%
#   column_spec(1:5, width = "3cm") %>%
#   kable_styling(latex_options = c("hold_position"))
#                 
                
kable(data, format = "latex", booktabs = TRUE) %>%
  column_spec(1:5, width = "4cm") %>%
  kable_styling(font_size = 9, latex_options = c("scale_down", "hold_position"))
```


## Honorary Research Fellow, University of Manchester (2021-present)

-   Collaboration on *MRC* programme grant with Prof. Judith Allen. Pleural Macrophage biology.

# Past Employment History
```{r}
emp <- read_excel("../data/employment.xlsx") 
emp <- emp[-c(1:2),] # remove descriptive title
emp %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

# Education
```{r}
edu <- read_excel("../data/education.xlsx") 
edu <- edu[-c(1:2),] # remove descriptive title
cols <- colnames(edu)
edu  %>% arrange(desc(when)) %>%  detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

# Funding

## Awarded
```{r}

gra <- read_excel("../data/grants.xlsx")
gra<- gra[-c(1:2),] # remove descriptive title
gra %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```
## Under review
2025 Irish Cancer Society Seed award (50K)

# Presentations, Awards & contributions

## Oral Presentations

```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what != 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```

## Poster Presentations

```{r}
pres <- read_excel("../data/presentations.xlsx")
pres<- pres[-c(1:2),] # remove descriptive title
pres  %>% arrange(desc(when)) %>% filter(what == 'Poster presentation') %>% brief_entries(
              what = with,# middle
              when = when,
              with = where)
```

## Media and Commentary

* [Commentary in *Immunity*: "Macrophages show up in style when Th2 lymphocytes organize their homecoming"](https://www.cell.com/immunity/fulltext/S1074-7613(23)00175-9)
* [Paper featured on The Immunology Podcast - Episode 51](https://www.immunologypodcast.com/ep-51-the-journey-of-cells-featuring-dr-doug-green)
* Immunity paper featured on August 2023 edition of *Journal of Allergy and Clinical Immunology* News Beyond Our Pages column.
* [Irish Health Professional 2023](https://www.irishhealthpro.com/content/articles/show/name/irish-led-research-tackles-parasitic-worms-that-infect-millions)
* [Medical Express 2023 ](https://medicalxpress.com/news/2023-03-tackle-parasitic-worms-infect-millions.html)


## Awards

```{r}
awards <- read_excel("../data/awards.xlsx")
awards<- awards[-c(1:2),] # remove descriptive title
awards %>% arrange(desc(when))  %>% brief_entries(
              what = with ,# middle
              when = when  # left
    ) # right
```



<!-- ## Oral Presentations (details of 8X poster presentations availible in appendex) -->

```{r}
# pres <- read_excel("../data/presentations.xlsx")
# pres<- pres[-c(1:2),] # remove descriptive title
# pres  %>% arrange(desc(when)) %>% filter(what != 'Poster presentation') %>% brief_entries(
#               what = with,# middle
#               when = when,
#               with = where)
```

<!-- ## Awards -->

```{r}
# awards <- read_excel("../data/awards.xlsx")
# awards<- awards[-c(1:2),] # remove descriptive title
# awards %>% arrange(desc(when))  %>% brief_entries(
#               what = with ,# middle
#               when = when  # left
#     ) # right
```

## Discipline Contributions
```{r}
rev <- read_excel("../data/reviewing.xlsx")
rev<- rev[-c(1:2),] # remove descriptive title
rev  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,
              with = with
                # left
    ) # right
```

# Active Collaborations

*Past Collaborations listed in Publications* 
```{r}
#short version
  sup <- read_excel("../data/collaborations.xlsx")

sup <- sup %>% filter(Active == "Y")
sup<- sup[-c(1:2),] # remove descriptive title
sup  %>% arrange(desc(when)) %>% detailed_entries(
  #what = what, #subtitle
                         #when = when, # to right
                         what = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```
<!-- \newpage -->

```{r}
# #long versions  
# sup <- read_excel("../data/collaborations.xlsx")
# sup<- sup[-c(1:2),] # remove descriptive title
# sup  %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
#                          when = when, # to right
#                          why = why , # sub-subtitle
#                          with = with, # Major title
#                          where =  where) #Rightmost)

```


# Teaching & Supervision

## Formal teaching
```{r}
#teaching short version 
# t <- read_excel("../data/teaching.xlsx")
# t<- t[-c(1:2),] # remove descriptive title
# t  %>% arrange(desc(when)) %>% brief_entries(
#               with= what ,# middle
#               when = when,
#               what = with# left
#     ) # right

t <- read_excel("../data/teaching.xlsx")



t %>% arrange(desc(when))  %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                         why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)

```

```{r}
# #teaching long version 
# t <- read_excel("../data/teaching.xlsx")
# t<- t[-c(1:2),] # remove descriptive title
# t  %>% arrange(desc(when)) %>% brief_entries(
#               with= what ,# middle
#               when = when,
#               what = with# left
#     ) # right

```

## Research Project Supervision
```{r}
# for longer version of CV
# sup <- read_excel("../data/supervison.xlsx")
# sup<- sup[-c(1:2),] # remove descriptive title
# sup  %>% arrange(desc(when)) %>% detailed_entries(what = what, #subtitle
#                          when = when, # to right
#                          why = why , # sub-subtitle
#                          with = with, # Major title
#                          where =  where) #Rightmost)
```

```{r}
sup <- read_excel("../data/supervison.xlsx")
sup<- sup[-c(1:2),] # remove descriptive title
sup$where <- sup$where %>% as.factor() %>% fct_recode("TCD"="Trinity College Dublin",
                                                                        "UoM" = "University of Manchester")

sup  %>% arrange(desc(when))  %>% brief_entries(
              what = what,# middle
              when = when, # left
              with = where # right
                # left
    ) # right
```

**Additional teaching duties** In 2016 I took on the academic duties for a Professor in TCD on teaching sabbatical. This included all aspects of the role, including practical supervision, assessment and marking, tutorials and administrative duties. I have organised visits for school students between 2013-2016. I sit on thesis committees for 3 PhD students.

# Outreach, organisation and PPI

**Discover Research Night**, Dublin Sep 2014. Marie Skłodowska-Curie-funded. I was the event organiser responsible for public engagement at Trinity Biomedical Sciences Institute, organising a team of 50 researchers in leading themed tours of scientific facilities. We were the most subscribed event of Dublin Research Night (\>500 members of the public).

**10th International Symposium on Bordetella**, Sep 2013. Logistics manager responsible for operations of the conference including registration, AV equipment and support for speakers.

**Manchester Immunology Group Seminar Series**, May 2017 -- Sep 2018. Co-lead organiser. Handling invitations, hospitality, communications, for renowned international speakers to visit University of Manchester.

**School talk**, Oaklands Community College, Edenderry Co. Offaly, 2021. "The life of a scientist".

 **Trinity Omics Core facility**. I have established formal pipelines for a single cell RNA-sequencing core service ‘Trinity Omics’. I have successfully lobbied my institute to fund a research assistant to support the service under my supervision. 
 
**Patient involvement** I have worked alongside patient representatives for Vasculitis Ireland as part of the PARADISE project. I have pitched an idea to the St James hospital Patient Representative Group and received feedback on development of a research programme, lay abstract and patient information leaflet.
 
 
<!-- \newpage -->


# Publications
<sup>*</sup> (Co-)corresponding author or (co-)senior authorship
```{r}
library(dplyr)
#knitr::write_bib(c("vitae", "tibble"), "My collection.bib")

#bibliography_entries("My collection.bib") %>%
  bibliography_entries("My Collection_zotero.bib") %>%
  arrange(desc(issued))

```
## Citation metrics

```{r fig.height=2, fig.width=8, dpi=300, dev='cairo_pdf'}
p1<- ggplot(ct, aes(year, cites)) +
    geom_segment(aes(xend = year, yend = 0), size=1, color='darkgrey') +
  ylab('Citations') +
  labs(caption = "Citations per year. Source: Google Scholar") +
   annotate("text", x=2013, y=150, label= paste('h-index:', l$h_index)) +
     annotate("text", x=2013, y=120, label= paste('i10-index:', l$i10_index)) +
  scale_x_continuous(breaks = c(2010:2025)) +
  geom_point(size=3, color= col) +
  theme_cowplot(font_size = 10) +
  theme(axis.title.x = element_blank(),
        axis.text.x = element_text(angle = 45, hjust = 1),
         plot.caption = element_text(face = "italic"))

# p1
papers <- read_excel("../data/Publication details.xlsx")

mytheme <- gridExtra::ttheme_default(
    core = list(fg_params=list(cex = 0.5)),
    colhead = list(fg_params=list(cex = 0.6)),
    rowhead = list(fg_params=list(cex = 0.6)))

p2 <- tableGrob(papers,theme = mytheme,rows = NULL)


grid.arrange(p1, p2,
             nrow=1,
             widths = c(0.4, 0.6),  # Adjust plot vs table width ratio
             as.table=TRUE)
```



# Referees
```{r}
ref <- read_excel("../data/referees.xlsx")
ref<- ref[-c(1:2),] # remove descriptive title
ref %>% detailed_entries(what = what, #subtitle
                         when = when, # to right
                        # why = why , # sub-subtitle
                         with = with, # Major title
                         where =  where) #Rightmost)
```

<!-- \newpage -->

\newpage

# Appendix

## Skills

I have a broad skillset that includes animal research, translational research, molecular biology, bioinformatics and clinical data analysis that enables me to design and manage research projects from a multidisciplinary point of view.

 | Category | Primary Skills | Tools & Technologies |
|----------|---------------|---------------------|
| Research & Analysis | Translational Research • Clinical Data Analysis • Animal Models • Patient Stratification | REDCap • Mass Cytometry • Flow Cytometry (30+ parameters) • ELISA |
| Programming | R (Fluent) • Python (Intermediate) • Unix/Bash (Basic) | RMarkdown • Git • HPC |
| Bioinformatics | Single Cell RNA-seq • Bulk RNA-seq • Mass Cytometry Analysis • Trajectory Analysis | Seurat • DESeq2 • limma • SCENIC • Spectre |
| Molecular Biology | NGS Library Prep • DNA/RNA Workflows • Proteomics • Cell Culture | Illumina NovaSeq • 10X Genomics • BD Rhapsody • TapeStation |
| Laboratory | Cell Sorting • Tissue Processing • Blood Processing • Core Facility Management | BD FACSAria • Mass Cytometry • Multiparameter Flow |

<!-- **Translational research and clinical data analysis**: Ability to manage and interact with multidisciplinary teams of clinicians, scientists, biobanking teams, biostatisticians and bioinformaticians. Experience with biobanking, clinical database management (RedCap).  Patient stratification, data curating and multi-modal integration: data analysis of longitudinal clinical datasets with 100s of clinical fields and 10000s of patients. -->

<!-- **Programming, Data analysis, Statistics & Visualisation**: R (Fluent, 7 years), Python (Intermediate, 5 years), RMarkdown (Intermediate, 5 years) Unix/Bash (Beginner). Data analysis/visualisation in R. Other programs: FlowJo and Prism and Adobe Illustrator. Database creation and maintenance. R Shiny Apps. high performance computing (basic). -->

<!-- **Bioinformatics** Expert in summarising take-home messages from single cell RNA-sequencing analysis in R and python (7 years experience). Mass cytometry analysis (FLowCore, Spectre). Analysis of microarray/bulk RNA-seq datasets analysis in R (limma, DESeq2), nCounter nanostring analysis. Pathway analysis (IPA, GO), regulatory network analysis (SCENIC), trajectory inference (Slingshot, RNA velocity), clustering (hierarchical and graph-based methods), dimension reduction (UMAP, Trimap, PCA), basic experience with machine learning (non linear, random forest) and systems biology (graphs, networks, logical modelling and differential equations). -->

<!-- **Animal models of Disease**: 12 years experience in *in vivo* model design: Autoimmunity (Experimental autoimmune encephalomyelitis, DSS-colitis, Imiquimod-induced inflammation), Cancer (B16 melanoma, CT26 lung), Infection (*L. sigmodontis*,*B. pertussis*, *F. hepatica*), Allergy (Allergen airway hypersensitivity) and general in vivo manipulation (Cell transfers, Irradiation/bone marrow transplantation peritonitis models, circadian rhythm modulation, cytokine administration, intrapleural injections). -->

<!-- **Molecular biology** NGS library preparation (Illumina) including single cell RNA-sequencing (BD Rhapsody and 10X genomics), Illumina sequencing (Novaaseq 6000)  sample preparation, study design for NGS, DNA & RNA extraction, RT-qPCR, nucleic acid QC (TapeStation, Qubit), PCR. Label free quantitative proteomics, bulk RNA-seq and ATAC-seq library preparation, enzyme assays. FFPE nucleic acid isolation. -->

<!-- **Laboratory techniques**: Expert in cell culture, cell sorting (BD Machines), high parameter (30+) flow cytometry, mass cytometry (including panel design, optimisation, storage, batch correction), tissue digestion, ELISA, cell isolation from blood (Whole blood, granulocytes/neutrophils, PBMC), lab management, inventory, ordering, health and safety. -->


## Datasets, code & websites 

```{r}
dat <- read_excel("../data/datasets.xlsx")
dat<- dat[-c(1:2),] # remove descriptive title
#pres  %>% arrange(desc(when)) %>% filter(what != 'Poster presentation') %>% brief_entries(

dat  %>% arrange(desc(when)) %>% detailed_entries(
              with = with,# Major title
              when = when, # to right
              what = why, # sub-subtitle
              where = where) #Rightmost)

```


<!-- ## Manuscripts in Preparation or review, etc (pre-prints listed as part of publications) -->

<!-- * Ridge, R.,  Finlay, C.M., O’ Farrelly, C., Moran, B., Alvarado-Vazquez, A., Hallgren, J., Conlon. CD34+ FcεR1+ mast cell precursors are increased in chronic spontaneous urticaria and predict clinical response to anti-IgE therapy -->

<!-- * Dwivedi, A., Ui Mhaonaigh, A., Carrol, M., Little, M and Finlay, C.M. Characterisation of low density neutrophils in SARS-Cov 2 patients (Under review) -->

<!-- * Finlay, C.M. Parkinson, J.P., Chan, B.H.K. and Allen, J.E. The proteome of the pleural fluid -->

<!-- * O'Conluain, R., Hollingsworth, s., Little, M. and Finlay, C.M., Soluble urokinase plasminogen activator receptor is a biomarker of kidney disease in ANCA-associated vasculitis -->


## Short-listed Applications

-   Short-listed for UKRI Innovation/Rutherford Fund Skills Development Fellowship (Computational Biology), 2017.

-   Short-listed for academic-industry partnership post-doctoral position, Belgium, 2019

-   Short-listed as candidate from school of medicine for Science Foundation Ireland Pathways programme, 2021

-   Short-listed for Lister Prize 2023

## Training

```{r}
train <- read_excel("../data/training.xlsx")
#train<- train[-c(1:2),] # remove descriptive title
train  %>% arrange(desc(Year)) %>% brief_entries(
              what = Course,# middle
              when = Year,# left
              with = Insitution# right
    ) 
```

## Thesis Committees and Continuation Vivas

```{r}
tc <- read_excel("../data/Thesis_committee.xlsx")
tc<- tc[-c(1),] # remove descriptive title
tc  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,# left
              with = with# right
    ) 
```
## Interview Panels
```{r}
rev <- read_excel("../data/Interview panels.xlsx")
rev<- rev[-c(1:2),] # remove descriptive title
rev  %>% arrange(desc(when)) %>% brief_entries(
              what = what,# middle
              when = when,
              with = with
                # left
    ) # right
```

## Courses

```{r}

cor <- read_excel("../data/courses.xlsx")

TCD <- cor$`Trinity College Dublin`
Manchester <- cor$`University of Manchester`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("University of Manchester",length(Manchester)),
          rep("Trinity College Dublin", length(TCD))), 
              detail = c(Manchester,TCD))
classes %>%
  detailed_entries(
    what = title,
    why = detail, .protect = F
  )
```


## Societies and Committees

```{r}

soc <- read_excel("../data/societies.xlsx")

socs <- soc$soc
com <- soc$`TCD Committees`
#external <- cor$Other

classes <- tibble(
  title =c(
          rep("Society Membership",length(socs)),
          rep("College Comittees", length(com))), 
              detail = c(socs,com))
classes %>%
  detailed_entries(
    what = title,
    why = detail#, .protect = F
  )
```







```{r eval=FALSE}
tribble(
  ~Year, ~Type, ~Desc,
  1903, "Physics", "Awarded for her work on radioactivity with Pierre Curie and Henri Becquerel",
  1911, "Chemistry", "Awarded for the discovery of radium and polonium"
) %>% 
  brief_entries(
    glue::glue("Nobel Prize in {Type}"),
    Year, 
    Desc
  )
```

```{r eval=FALSE}
library(tibble)
tribble(
  ~ Degree, ~ Year, ~ Institution, ~ Where,
  "Informal studies", "1889-91", "Flying University", "Warsaw, Poland",
  "Master of Physics", "1893", "Sorbonne Université", "Paris, France",
  "Master of Mathematics", "1894", "Sorbonne Université", "Paris, France"
) %>% 
  detailed_entries(Degree, Year, Institution, Where)
```
